modules/claims_api/spec/support/rswag_config.rb

```diff
@@ -117,6 +117,10 @@ def config # rubocop:disable Metrics/MethodLength
             description: <<~VERBIAGE
               Allows authenticated and authorized users to automatically establish an Intent to File (21-0966) in VBMS.
             VERBIAGE
+          },
+          {
+            name: 'Disability',
+            description: 'Allows authenticated and authorized users to file a Disability Claim (21-526EZ) in VBMS.'
```

Comment:
this line gets added to both the dev and production swagger file by default, right? I'm assuming you're manually removing this from the production/swagger? if so, I'd vote to not include this for now. it just adds the little tag line on the dev portal, and we'd have to remember to manually remove it after each run of rswag. we'll be fine just seeing the actual 526 endpoint docs in dev / staging... 

probably a bit confusing to read here. I'll send a photo via slack to give you reference.

Reply:
Removed from config, rebuilt swagger, and pushed up.